scripts/zones/La_Theine_Plateau/npcs/Vicorpasse.lua

```diff
@@ -27,7 +27,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 4) then
+		if(MissionStatus >= 0 and MissionStatus <= 4) then

```

Comment:
ooops, i just made some new chars and checked in retial today. /cry >.>
btw it's easy to miss Galaihaurat and hence has to go back up that hill to click him for the "bronze sword", not the beginingg.